llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -462,8 +461,6 @@ InstrItinData<II_MOVA_M, [SimpleCycle<M_WM_PORT>], [1,1]>,
 InstrItinData<II_MOVA_P, [SimpleCycle<P_WM_PORT>], [1,1]>,
 InstrItinData<II_MOVA_R, [SimpleCycle<R_WA_PORT>], [1,1]>,
 InstrItinData<II_MOVX, [SimpleCycle<R_WX_PORT>], [1,1]>,
-InstrItinData<II_MOVXM, [PrefixCycle<P_WM_PORT>, PrefixCycle<S_WM_PORT>, PrefixCycle<M_WM_PORT>, PrefixCycle<DJ_WM_PORT>,
-                         PrefixCycle<DN_WM_PORT>, PrefixCycle<DC_WM_PORT>, SimpleCycle<R_WM_PORT>], [1,1]>,
```

Comment:
MOVX and MOVXM seem to be on different write ports, X vs M. How can MOVX subsume all of MOVXM?

Reply:
Hi @martien-de-jong 

When I started following was state of Schedule for MOV & MOVXM (https://github.com/Xilinx/llvm-aie/blob/d74d6a0f993eb66667004e313f44f1c15c041609/llvm/lib/Target/AIE/AIE2Schedule.td)
`InstrItinData<II_MOV     , [PrefixCycle<P_WM_PORT>, PrefixCycle<S_WM_PORT>, SimpleCycle<R_WM_PORT>], [1,1]>,`
`InstrItinData<II_MOVXM, [PrefixCycle<P_WM_PORT>, PrefixCycle<S_WM_PORT>, SimpleCycle<R_WM_PORT>], [1,1]>,`